.github/workflows/scripts/scanner-update-offline-bundle.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+
+SCANNER_V4_DEFS_BUCKET="https://storage.googleapis.com/scanner-v4-test"
+ROX_PRODUCT_VERSION="$1"
+PRODUCT_VERSION="${ROX_PRODUCT_VERSION}.0"
+declare -A files_to_download=(
+    ["v4/vulns.json.zst"]="${SCANNER_V4_DEFS_BUCKET}/vulnerability-bundles/${PRODUCT_VERSION}/vulns.json.zst"
+    ["v4/repository-to-cpe.json"]="${SCANNER_V4_DEFS_BUCKET}/redhat-repository-mappings/repository-to-cpe.json"
+    ["v4/container-name-repos-map.json"]="${SCANNER_V4_DEFS_BUCKET}/redhat-repository-mappings/container-name-repos-map.json"
```

Comment:
Just noticed that these files are not updated/output from the `Scanner release vulnerability update` job - think these files need to be pulled out of the `mapping.zip` in the same location.